Cabal/doc/nix-local-build.rst

```diff
@@ -362,8 +362,15 @@ cabal new-repl
 --------------
 
 ``cabal new-repl TARGET`` loads all of the modules of the target into
-GHCi as interpreted bytecode. It takes the same flags as
-``cabal new-build``.
+GHCi as interpreted bytecode. In addition to the ``cabal new-build``'s flags,
+it takes an additional ``--repl-options`` flags.
+
+To avoid ``ghci`` specific flags from triggering unneeded global rebuilds these
+flags are now stripped from ``ghc-options``. As a result ``--ghc-options`` will
```

Comment:
I got momentarily confused here between ``ghc-options`` the Cabal file field, and ``--ghc-options`` the command-line argument, which is what are being talked about here, right? Worth clarifying.

Reply:
To be honest, it refers to both. ``--ghc-options`` the flag and ``ghc-options`` the field end up in the same data structure that gets filtered, so this applies in both cases.